lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
```

Comment:
@pchaigno, while we’re at it — I was curious about the `group` field. I haven’t included it since we want to make Starlark a dedicated language. Is it OK? For example, Sage includes it — but Sage files are Python files 1:1 (while Starlark isn’t). Mostly asking for hidden cases I’m not aware of.

Reply:
It's used to group languages in statistics. For example, if language A has `group: B`, all files of language A will be counted toward language B in statistics. They can still receive a syntax highlighting specific to language A and will be displayed separately in searches. We increasingly use this to apply different syntax highlighting grammars to different dialects of a single language.